crates/net/discv4/src/lib.rs

```diff
@@ -1953,6 +1999,9 @@ struct LookupContextInner {
     /// The closest nodes
     closest_nodes: RefCell<BTreeMap<Distance, QueryNode>>,
     /// A listener for all the nodes retrieved in this lookup
+    ///
+    /// This is present if the lookup was triggered manually via [Discv4] and we want to return all
```

Comment:
```suggestion
    /// This is present if the lookup was triggered manually via [`Discv4`] and we want to return all
```

Reply:
this is not required

Reply:
the standard library does it, it's definitely smthg we want to aim for https://doc.rust-lang.org/src/alloc/vec/mod.rs.html#44

Reply:
both are supported `[Discv4]` is newer afaik.
the codebase mostly uses this variant for intra doc link.
no need for the extra `